src/models/stores.ts

```diff
@@ -29,17 +30,13 @@ export function createStores(params: ICreateStores, authoring: any): IStores {
     ui: params && params.ui || createUIModel(authoring.spaces),
     populations: params && params.populations ||
       createPopulationsModel(authoring.curriculum, flatten(authoring.populations)),
-    backpack: params && params.backpack || BackpackModel.create({}),
-    cellZoom: params && params.cellZoom
-      || CellZoomModel.create({
-           organisms: {
-            one: CellMouseModel.create({ id: "one", baseColor: "brown" }),
-            two: CellMouseModel.create({ id: "two", baseColor: "white" })
-          },
-          rows: [
-            CellZoomRowModel.create({organism: "one"}),
-            CellZoomRowModel.create({organism: "two"}),
-          ]
-         })
+    backpack: params && params.backpack
```

Comment:
I don't think this is needed now that the populations-backpack feature is in, as the user can now add their own mice to the backpack.

Reply:
I figured we might want the ability to show default mice at some point, so I left it for now

Reply:
But if we merge this in there will be no way to open a model without the two collected mice.